docs/sts/assume-role.go

```diff
@@ -125,6 +131,18 @@ func main() {
 		return
 	}
 
+	// API requests are secure (HTTPS) if secure=true and insecure (HTTP) otherwise.
+	// New returns an MinIO Admin client object.
+	madmClnt, err := madmin.NewWithOptions(stsEndpointURL.Host, mopts)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	err = madmClnt.ServiceRestart(context.Background())
+	if err != nil {
+		log.Fatalln(err)
+	}
```

Comment:
This is more like a testing logic - do we want to add this to the example/doc?

Reply:
yeah the idea is to show that after a restart() of the service the account is still active, "immediately"

Reply:
basically also ends up showing how same STS can be used for admin operations as well. I wanted to add this to the test itself, but couldn't finish it in time as had to work on other items.